Utils/Dataflow/README

```diff
@@ -91,6 +91,30 @@ intended for making sure that information is consistent between ProdSys2 and
 ES. It gets a very basic set of metadata from ProdSys2, adds ES-related
 fields, and checks that it is present in ES rather than uploading it.
 
+================
+* Stages types *
+================
+There are three types of stages:
+
+1. Source stage - an initial stage in a dataflow pipeline that gets data
+from an external source (CDS, Rucio, AMI, etc.)
```

Comment:
Missing period at the end of sentence: "... AMI, etc.).". I had my doubts about this, but it seems that this particular case have already been discussed: https://english.stackexchange.com/questions/23022/how-to-deal-with-abbreviations-like-etc-at-the-end-of-parentheses-which-are-c.

Reply:
According to the discussion:
>The dot in etc. If it's inside the parentheses, like in this case, then you should use two dots, since one is for the abbreviation, the other is for the sentence.

Fixed. Now it looks like you've described.